plugins/proxqp/QpSolversEigenProxqp.cpp

```diff
@@ -582,6 +588,65 @@ bool ProxqpSolver::setStringParameter(const std::string& parameterName, const st
     return false;
 }
 
+bool ProxqpSolver::getBooleanParametersNames(std::vector<std::string>& parametersNames) const
+{
+    parametersNames.resize(0);
```

Comment:
Nit: why not ``clear()``?

Reply:
Because I did not remember it existed. `clear()` is clearly better, I will change it.